README.qmd

```diff
@@ -130,7 +126,7 @@ distritos_wgs84 <- distritos |>
 plot(sf::st_geometry(distritos_wgs84))
 ```
 
-## Estudios basicos
+## Estudios básicos
```

Comment:
:+1: although what does that even mean?

Would be happy to change this section title.

Reply:
I agree.